documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/main.cadl

```diff
@@ -0,0 +1,60 @@
+import "@cadl-lang/rest";
+import "@cadl-lang/versioning";
+import "@azure-tools/cadl-autorest";
```

Comment:
I don't think this file is using the autorest extension at all. We should not import thing we don't use, especially autorest should be limited.